src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/Transform.cs

```diff
@@ -204,7 +204,7 @@ public Hashtable PropagatedNamespaces
                 CanonicalXmlNodeList namespaces = null;
                 if (reference != null)
                     namespaces = reference._namespaces;
-                else if (signedXml._context != null)
+                else if (signedXml?._context != null)
```

Comment:
@krwq I'm pretty sure that our normal tactic here is to always have netfx-impacting changes be as small as possible; so this should stay broken for now, and get fixed as a separate issue.

@AlexGhiondea: Thoughts?